azure-defender-easm/scripts/kusto_queries.py

```diff
@@ -0,0 +1,108 @@
+from kusto_client import KUSTO_CLIENT, KUSTO_DATABASE
+from azure.kusto.data.helpers import dataframe_from_result_table
+
+
+def get_host_asset(host_name):
+    query = f"EasmHostAsset | where Host == '{host_name}' | limit 1"
```

Comment:
Should look into using query parameters for KQL injection protection:
https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/query-parameters-statement?pivots=azuredataexplorer#example